
S.th. modified is not const ... for gcc-3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./mount/mount.c.vanilla	2004-07-04 20:35:50.000000000 +0200
+++ ./mount/mount.c	2004-07-04 20:36:29.000000000 +0200
@@ -1104,7 +1104,7 @@
  * Return 0 for success (either mounted sth or -a and NOAUTO was given)
  */
 static int
-mount_one (const char *spec, const char *node, char *types, const char *opts,
+mount_one (const char *spec, char *node, char *types, const char *opts,
 	   char *cmdlineopts, int freq, int pass) {
   int status;
   int status2;
